app.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
```

Comment:
Since we're using es6, we can use `const` for modules
> var express = require('express');

const express = require('express');